integration/release/error.go

```diff
@@ -1,8 +1,8 @@
-package basic
+// +build k8srequired
```

Comment:
Was this just missing before? Don't know much about k8s build tags.

Reply:
Yes this was missing. Not a big deal but we should have the tag in our e2e tests.